### ISSUE TITLE
Add an option to strip text inside html tags

### ISSUE THREAD

**Issue by Tratoschek** [Alir3z4/html2text#142]
It would be nice, if you could add an option to strip text inside html tags. Sometimes poorly formatted webpages provide such text.
- Version by `pyhtml2text --version`: pyhtml2text 2016.5.29
- Test script

```
>>> import html2text
>>> html2text.html2text('<b>text </b>')
'**text **\n\n'
```

Much better would be `'**text**\n\n'`
- Python version `python --version`: Python 3.4.3

**Comment by theSage21** [comment]
This seems reasonable. I'll get to it in a while if that's ok. In case you want to you could submit a PR for this.

**Comment by jonathan-s** [comment]
Fixed by #177
